src/components/side-panel/panel-toggle.tsx

```diff
@@ -40,15 +40,17 @@ const PanelHeaderBottom = styled.div.attrs({
 `;
 
 PanelToggleFactory.deps = [PanelTabFactory];
-function PanelToggleFactory(PanelTab) {
-  const PanelToggle = ({activePanel, panels, togglePanel}) => {
+
+function PanelToggleFactory(PanelTab: ReturnType<typeof PanelTabFactory>) {
+  const PanelToggle = ({activePanel, panels, togglePanel}: PanelToggleProps) => {
```

Comment:
to be consistent as other component
`const PanelToggle = React.FC<PanelToggleProps>`

Reply:
fixed